src/mapport.cpp

```diff
@@ -119,28 +117,13 @@ static bool ProcessPCP()
 
     // We don't delete the mappings when the thread is interrupted because this would add additional complexity, so
     // we rather just choose a fairly short expiry time.
```

Comment:
This comment seems to be lacking context now.

Reply:
Why? If we are interrupted the `while()` clause above will continue here and we won't tell the router to remove the mappings and just return instead.